### ISSUE TITLE
Typo in documentation

### ISSUE THREAD

**Issue by isik-kaplan** [graphql-python/graphene-django#1052]
* **Other information** 
https://docs.graphene-python.org/projects/django/en/latest/queries/#resolvers
```python
import graphene
from .models import Question
from .types import QuestionType

class Query(graphene.ObjectType):
    foo = graphene.List(QuestionType)

    def resolve_foo(root, info):
        id = kwargs.get("id")
        return Question.objects.get(id)
```

Missing `kwargs` in function definition and `id` kwarg in return statement. I think there are couple more small typos like this in the documentation, I'll add them to this issue when I stumble upon them.

**Comment by rganeyev** [comment]
Fixed in #1078, please review

**Comment by rganeyev** [comment]
thanks for merging #1078, feel free to close the issue
